docs/tutorials/tictoc/part1.md

```diff
@@ -101,7 +101,7 @@ registered in OMNeT++ with the `Define_Module()` macro.
 
 !!! note
     It is a common mistake to forget the `Define_Module()` line. If it is missing,
-    you'll get an error message similar to this one: `"Error: Class 'Txc1' not found -- perhaps
+    you'll get an error message similar to this one: `"Error: Class 'Txc1' not found -- perhaps 
```

Comment:
Spaces at the ends of lines are not needed, the line break suffices - given HTML rules, and maybe of other processing before that.